src/webui/components/creator/elementary/TimeElapsedColumn.tsx

```diff
@@ -1,20 +1,17 @@
 
-import { ClickArea, Column, Text } from '^/webui/designsystem/module';
+import { Column, Text } from '^/webui/designsystem/module';
 
 import TimeElapsed from '../../common/TimeElapsed';
 
 export type Props = {
     fullName: string;
     date: Date;
-    profileHandler: () => void;
 };
 
-export default function Component({ fullName, date, profileHandler }: Props)
+export default function Component({ fullName, date }: Props)
 {
     return <Column alignY='justify' gap='none'>
-        <ClickArea clickHandler={profileHandler}>
```

Comment:
Again, I think we need to restore this because we've lost an important piece of functionality.